src/api/xxAMIDOxx.xxSTACKSxx.Infrastructure/DependencyRegistration.cs

```diff
@@ -15,6 +15,8 @@
 using xxAMIDOxx.xxSTACKSxx.Infrastructure.HealthChecks;
 using Amido.Stacks.DynamoDB.Extensions;
 using Amazon.DynamoDBv2;
+using Amido.Stacks.SQS;
```

Comment:
I've been thinking it would be good to wrap the using statements in the preprocessor flags as well to reduce the amount of things people will need to remove after.